### DISCUSSION TITLE
[Typography plugin]: table > caption is not styled

### DISCUSSION THREAD

**Discussion by tordans** [tailwindlabs/tailwindcss#7603] [General]
### What version of @tailwindcss/typography are you using?

0.4.1

### What version of Node.js are you using?

16

### What browser are you using?

chrome

### What operating system are you using?

macOS

### Reproduction repository

https://supaplexosm.github.io/strassenraumkarte-neukoelln/parkraumkarte/report#but_not_published_ATM

### Describe your issue

The caption element of a table is not styles ATM.
However, IMO it should be styled like title of the table.

> The <caption> HTML element specifies the caption (**or title**) of a table.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption

Here are before/after screenshots with custom styling in place.
Note, that I have to use the TailwindCSS JIT `!`-utility classes to get the styles applied at all.

<img width="1141" alt="table-captoin--before" src="https://user-images.githubusercontent.com/111561/133923874-ae32cc18-29bb-46f6-86c9-7bb6ec2e7022.png">

<img width="1144" alt="table-caption--after" src="https://user-images.githubusercontent.com/111561/133923870-07b093ed-c48a-4d38-bf6e-b9528e8b6c29.png">

**Comment by simonswiss** [comment]
Hey! Like I mentioned in my [other response](https://github.com/tailwindlabs/tailwindcss-typography/issues/200#issuecomment-1048401635), the Typography plugin does not cater for many elements within a `table`.

I am not saying that `table > caption` has no valid reason to be styled - it might be something we've overlooked, but in cases like these where you're trying to style an element that has no default styling from the plugin, you can either use utility classes directly on the elements (whenever possible), or customize the CSS in your config file:

https://tailwindcss.com/docs/typography-plugin#customizing-the-css

Moving this to our GitHub discussions as it's not necessarily an _issue_ with the plugin, rather a feature suggestion.
